### PR TITLE
PostgresQueryBuilder

### PR DESCRIPTION
**Pull request by AllenX2018**
#4171 
add support for postgresql querybuilder

### CODE REVIEW

**Review by flycash — COMMENTED** [review]
Please add unit tests. I will add CI for postgress DB

File: pkg/client/orm/qb_postgres.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
+package orm
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var Quote string = `"`
```
Comment (zouyx) [inline pkg/client/orm/qb_postgres.go:9]: private variable or not? if private , change scope pls

File: pkg/client/orm/qb_postgres.go [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+// PostgresQueryBuilder is the SQL build
+type PostgresQueryBuilder struct {
+	Tokens []string
```
Comment (zouyx) [inline pkg/client/orm/qb_postgres.go:13]: as above

File: pkg/client/orm/qb_postgres.go (hunk 1 above)
Comment (AllenX2018) [reply pkg/client/orm/qb_postgres.go:9]: ok

**Review by flycash — APPROVED** [review]

**Pull request merged**